packages/ra-core/src/form/FormDataConsumer.tsx

```diff
@@ -119,21 +117,10 @@ export const FormDataConsumerView: SFC<Props> = ({
     return ret === undefined ? null : ret;
 };
 
-const mapStateToProps = (
-    state: ReduxState,
-    { form, record }: ConnectedProps
-) => ({
-    formData: getFormValues(form)(state) || record,
-});
+const FormDataConsumer = (props: ConnectedProps) => {
```

Comment:
seeing how little this component brings, I suggest we deprecate it by removing it of the documentation.

Reply:
As discussed, we'll see that in another PR